src/main/plugins/org.dita.pdf2/cfg/common/vars/en.xml

```diff
@@ -29,6 +29,9 @@ with those set forth herein.
 This file is part of the DITA Open Toolkit project. 
 See the accompanying LICENSE file for applicable license.
 -->
+
+<!-- 20170503 SCH: Add support for troubleshooting elements. -->
+
```

Comment:
Hm. Saw similar entries in other files, so was following example (taskdisplay.xsl)
<!-- 20090904 RDA: Add support for stepsection; combine duplicated logic
                   for main steps and steps-unordered templates. -->

I will remove them.